### PR TITLE
Added wait till server quiesce for windows platform

### LINKED ISSUE #19138: Unable to find the output in the server log
Despite the fact that "Unable to find the output in the server log" exception is created, searched String appears in the log later. In order to see PreDestroy message, a wait is added till application to stop.

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #19138

### CODE REVIEW

**Comment by BerksanAtes** [comment]
SOE PB link -> https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_kIHLwHo6Eeyxm-wYSNWOrg

**Comment by BerksanAtes** [comment]
#run-libby-bot

File: dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/endpoint/AbstractJaxWsWebEndpoint.java [hunk 1]
```diff
@@ -164,7 +167,19 @@ public void destroy() {
     public void invoke(HttpServletRequest request, HttpServletResponse response) throws ServletException {
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/endpoint/AbstractJaxWsWebEndpoint.java]: Would it be possible to just add final modifiers to the method signatures so you don't have instantiate new variables?

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/build.gradle [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+dependencies {
+  requiredLibs 'org.apache.wink:wink-client:1.4',
+    'org.apache.wink:wink-common:1.4',
```
Comment (neuwerk) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/build.gradle:14]: I don't think you need the JAX-RS dependencies now you've split the FATs.

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithoutUserBundle/server.xml.ee9 [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<server>
+    <featureManager>
+        <feature>pages-3.0</feature>
+        <!--  <feature>globalWebservicesHandler-1.0</feature> -->
+    </featureManager>
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <logging traceSpecification="*=info:logservice=all:app.manager=all:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all"/>
```
Comment (neuwerk) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithoutUserBundle/server.xml.ee9]: can you remove this trace line just to keep things cleaner?

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithoutUserBundle/server.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<server>
+    <featureManager>
+        <feature>jsp-2.3</feature>
+        <!--  <feature>globalWebservicesHandler-1.0</feature> -->
+    </featureManager>
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <logging traceSpecification="*=info:logservice=all:app.manager=all:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all"/>
```
Comment (neuwerk) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithoutUserBundle/server.xml]: can you remove this trace line just to keep things cleaner?

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithUserBundle/server.xml.ee9 [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<server>
+    <featureManager>
+        <feature>pages-3.0</feature>
+        <feature>usr:RSHandler1Feature</feature>
+        <!-- <feature>globalWebservicesHandler-1.0</feature> -->
+    </featureManager>
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <logging traceSpecification="*=info:logservice=all:app.manager=all:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all"/>
```
Comment (neuwerk) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithUserBundle/server.xml.ee9]: can you remove this trace line just to keep things cleaner?

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithUserBundle/server.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<server>
+    <featureManager>
+        <feature>jsp-2.3</feature>
+        <feature>usr:RSHandler1Feature</feature>
+        <!-- <feature>globalWebservicesHandler-1.0</feature> -->
+    </featureManager>
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <logging traceSpecification="*=info:logservice=all:app.manager=all:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all"/>
```
Comment (neuwerk) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/WithUserBundle/server.xml]: can you remove this trace line just to keep things cleaner?

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/Default/server.xml.ee9 [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<server>
+    <featureManager>
+        <feature>pages-3.0</feature>
+        <feature>restfulWS-3.0</feature>
+    </featureManager>
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <logging traceSpecification="*=info:logservice=all:app.manager=all"/>
```
Comment (neuwerk) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/Default/server.xml.ee9]: can you remove this trace line just to keep things cleaner?

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/Default/server.xml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<server>
+    <featureManager>
+        <feature>jsp-2.3</feature>
+        <feature>jaxrs-2.0</feature>
+    </featureManager>
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <logging traceSpecification="*=info:logservice=all:app.manager=all"/>
```
Comment (neuwerk) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/publish/files/GlobalHandlerFeatureOnly/Default/server.xml]: can you remove this trace line just to keep things cleaner?

**Review by neuwerk — CHANGES_REQUESTED** [review]
See comments above! Over all looks good, just some simple changes needed to clean things up a tiny bit.

File: dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/endpoint/AbstractJaxWsWebEndpoint.java (hunk 1 above)
Comment (BerksanAtes) [reply dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/endpoint/AbstractJaxWsWebEndpoint.java]: done

**Review by neuwerk — APPROVED** [review]
Changes look good! Thanks so much for doing this @BerksanAtes!!

**Review by jim-krueger — COMMENTED** [review]
I hate to even mention this (considering how many files are in this PR), but I believe you need to change the copyright dates to 2022.  Sorry

**Review by jim-krueger — APPROVED** [review]

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**